waspc/data/Generator/templates/react-app/vite.config.ts

```diff
@@ -29,12 +30,13 @@ const defaultViteConfig = {
     outDir: "build",
   },
   test: {
+    globals: true,
     environment: "jsdom",
-    setupFiles: ["./src/test/vitest/setup.ts"],
+    // vitest is running from the root of the project, so we need
+    // to specify the path to the setup file relative to the root.
+    setupFiles: {=& vitestSetupFilesArray =},
+    exclude: [...defaultExclude, ".wasp/**/*"]
```

Comment:
Same story as with setup files, `.wasp` is already defined somewhere in Haskell I think

Reply:
Good call, I've now changed the template 👍